### DISCUSSION TITLE
Suite of libraries?

### DISCUSSION THREAD

**Discussion by chmorgan** [chmorgan/sharppcap#325] [General]
Hey @kayoub5, @phyxionnl. Thoughts on moving sharppcap, PacketDotNet, and packetdotnet-connections under some higher level team/project to reflect that they are a suite of libraries that can work together?

It seems like people are unaware of packetdotnet-connections for example, and if we had a diagram showing the layers and what they did, sharppcap on capture/send, PacketDotNet on low level, connections leveraging PacketDotNet, it could be easier for people to follow.

And with you helping out and my desire for the libraries to live on it might make sense to shift them out from under my GitHub account both to reflect your participation and to decentralize management.

Thoughts?

**Comment by PhyxionNL** [comment]
Sounds good to me @chmorgan, we'd have to decide on a suitable name though 🙂

BTW, one problem with the connections library is the license, it's not as friendly as packetnet, which I think is main problem when you have two projects that belong so closely together.

**Reply by kayoub5** [reply]
Good proposal, no objections from my side.
Just make sure to update all links (including the ones in awesome dotnet).

**Comment by chmorgan** [comment]
Totally fine with adjusting the license to match. I think I’m the only one
with code in that library. We use mpl for packetdotnet net today right?

**Reply by PhyxionNL** [reply]
Yeah, MPL 2.0 👍

**Comment by chmorgan** [comment]
Any thoughts on a team/project name or whatever GitHub calls them?

**Reply by kayoub5** [reply]
it's called an organization, some choose to have the project name as organization name, like sharppcap/sharppcap

**Reply by PhyxionNL** [reply]
Maybe packetcapture (packetcap already exists) as a combination of packetdotnet and sharppcap? I don't really have any other ideas.

**Reply by chmorgan** [reply]
@kayoub5 any ideas on organization names? sharpcapture? packetcapture (from PhyxionNL)? dotcapture dotcap ?

**Reply by kayoub5** [reply]
dotcap => no, would confuse with libcap => Linux capability lib

I would go for one of below:
* having the org name "sharppcap" like https://github.com/reviewdog/reviewdog and https://github.com/danger/danger
* dotpcap (**dot**net **pcap** group)
* ironpcap some .NET libraries/group use the prefix iron when porting something to .NET, see https://github.com/search?l=C%23&o=desc&p=1&q=iron&s=stars&type=Repositories

others sound too long or not catchy enough.

**Comment by chmorgan** [comment]
We’d be rolling packetdotnet under this as well I think.

**Reply by chmorgan** [reply]
@PhyxionNL any comments on the proposed names?

**Reply by PhyxionNL** [reply]
I don't have a strong opinion on a name. I think it would be useful to group both projects under one organization for visibility, so a name that somewhat combines both sound like a good idea. dotpcap would be fine I guess.

**Comment by chmorgan** [comment]
@kayoub5, @PhyxionNL dotpcap work? If so I'll set up that org and move the projects over.

**Reply by kayoub5** [reply]
yup, sounds good

**Reply by PhyxionNL** [reply]
Sounds good to me.

**Comment by chmorgan** [comment]
@kayoub5 @PhyxionNL repositories have been transferred and you guys are part of the dotpcap organization.
